src/OpenLoco/src/LastMapWindowAttributes.h

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+#include <cstdint>
+#include "Window.h"
+
+namespace OpenLoco::LastMapWindow

```

Comment:
It is perhaps a bit overkill to put this in a namespace on its own. The map window is part of the UI. I would suggest putting it in the UI namespace.